### PR TITLE
feat: Add more logs to on-chain-relayer

### PR DESCRIPTION
**Pull request by koonopek**
sample:
```bash
Starting contract prices updater with interval 10000
Update condition NOT satisfied: Not enough time has passed to update prices: {"timeDiff":70145,"updatePriceInterval":110000}; Value has not deviated enough to be updated. {"VST":{"valueFromContract":0.99367395,"valuesFromNode":[0.9936990950421556],"packagesCount":1},"maxDeviation":"0.0025","thresholdDeviation":"100.0000"}
Update condition NOT satisfied: Not enough time has passed to update prices: {"timeDiff":80107,"updatePriceInterval":110000}; Value has not deviated enough to be updated. {"VST":{"valueFromContract":0.99367395,"valuesFromNode":[0.9937484136657652],"packagesCount":1},"maxDeviation":"0.0075","thresholdDeviation":"100.0000"}
Update condition NOT satisfied: Not enough time has passed to update prices: {"timeDiff":90166,"updatePriceInterval":110000}; Value has not deviated enough to be updated. {"VST":{"valueFromContract":0.99367395,"valuesFromNode":[0.9936959208665944],"packagesCount":1},"maxDeviation":"0.0022","thresholdDeviation":"100.0000"}
```

### CODE REVIEW

**Comment by koonopek** [comment]
I think that addding deviation per source will add too much text for the value it can be easly computed if needed.

File: packages/on-chain-relayer/src/run-relayer.ts [hunk 1]
```diff
@@ -66,7 +62,8 @@ const task = new AsyncTask(
 
 const job = new SimpleIntervalJob(
   { milliseconds: relayerIterationInterval, runImmediately: true },
-  task
+  task,
+  { preventOverrun: true }
```
Comment (hatskier) [inline packages/on-chain-relayer/src/run-relayer.ts:66]: 👍

File: packages/on-chain-relayer/src/core/update-conditions/value-deviation-condition.ts [hunk 2]
```diff
@@ -58,3 +71,56 @@ const calculateDeviation = (
 
   return (pricesDiff * 100) / valueFromContract;
 };
+
+class ValueDeviationLogTrace {
```
Comment (hatskier) [inline packages/on-chain-relayer/src/core/update-conditions/value-deviation-condition.ts:75]: Just curious, why did you decide to make a class for it? Couldn't we just collect all details in a simple object?

File: packages/on-chain-relayer/src/core/update-conditions/value-deviation-condition.ts [hunk 3]
```diff
@@ -58,3 +71,56 @@ const calculateDeviation = (
...
+      valuesFromNode: number[];
+      timestamp?: number;
+      packagesCount: number;
+    }
+  > = {};
+  private maxDeviation!: string;
+  private thresholdDeviation!: string;
+
+  addPerDataFeedLog(
+    dataFeedId: string,
```
Comment (cehali) [inline packages/on-chain-relayer/src/core/update-conditions/value-deviation-condition.ts:89]: `dataFeedId` can be taken from `dataPoint`

Comment (koonopek) [reply packages/on-chain-relayer/src/core/update-conditions/value-deviation-condition.ts]: Good point

**Review by hatskier — DISMISSED** [review]

**Review by hatskier — APPROVED** [review]

**Pull request merged**